asset/mainnet/app.toml

```diff
@@ -63,6 +63,10 @@ BEP153Height = 284376000
 #Block height of BEP173 upgrade
 BEP173Height = 284376000
 FixDoubleSignChainIdHeight = 9223372036854775807
+# Block height of BEP159Height upgrade
+BEP159Height = 298721375
+# Block height of LimitConsAddrUpdateInterval upgrade
+LimitConsAddrUpdateIntervalHeight = 298721375
```

Comment:
Enable two updates in the same height?

Reply:
Yes, because on testnet it is a different height.